ppr-ui/tests/unit/ConfirmDischarge.spec.ts

```diff
@@ -91,6 +92,8 @@ describe('ConfirmDischarge registration view', () => {
     expect(wrapper.findComponent(RegistrationFee).vm.registrationType).toBe('Total Discharge')
     // buttons
     expect(wrapper.findComponent(ButtonsStacked).exists()).toBe(true)
+    // folio
+    expect(wrapper.findComponent(FolioNumberSummary).exists()).toBe(true)
```

Comment:
can you add a test in here for when folio is invalid it blocks the submit?

Reply:
submit doesn't do anything right now.... so it's hard to tell ;)
I did put in a test for the show error property

Reply:
ah but it does